src/MR.AspNetCore.Jobs.PostgreSQL/JobsDbContextFactory.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+using Microsoft.Extensions.DependencyInjection;
+using MR.AspNetCore.Jobs.Models;
+
+namespace MR.AspNetCore.Jobs
+{
+	public class JobsDbContextFactory : IDesignTimeDbContextFactory<JobsDbContext>
+	{
+		internal static string DevConnectionString =
+			Environment.GetEnvironmentVariable("MR_ASPNETCORE_JOBS_POSTGRESQL_CS_DEV") ??
+			@"Server=127.0.0.1;Port=5432;Database=MR.AspNetCore.Jobs.Dev;User Id=postgres;Password=password;";
```

Comment:
FYI, this overrides what you configure as your connection string in your applications service configuration, so probably should remove it.

Reply:
Unless I'm mistaken somehow, this is the connection string that will be used at design time while developing only (to run migrations). In your own app, this whole class should never be used. That's supposedly what `IDesignTimeDbContextFactory` does. Can you explain why you think this will override that in the app?